R/derive_date_vars.R

```diff
@@ -1263,10 +1263,15 @@ compute_tmf <- function(dtc,
 #'
 #' @param flag_imputation Whether the date imputation flag must also be derived.
 #'
-#'   If `"auto"` is specified, the date imputation flag is derived if the
-#'   `date_imputation` argument is not null.
+#'   If `"auto"` is specified and `date_imputation` argument is not null or ,
+#'   then date imputation flag is derived.
+#'
+#'   If `"date"` is specified, then date imputation flag is derived.
+#'
+#'   If `"none"` is specified, then no date imputation flag is derived.
 #'
 #'   *Permitted Values*: `"auto"`, `"date"` or `"none"`
+#'   *Default*: `"auto"`
```

Comment:
Should be removed because we decided that the default value should not be stated in the argument description.